jobs/datadog-cc-worker/templates/process.yaml.erb

```diff
@@ -0,0 +1,6 @@
+init_config:
+
+instances:
+  - name: cc api worker
```

Comment:
There is more than one worker process, and it's controlled by the property [cc.jobs.local.number_of_workers](https://github.com/cloudfoundry/cloud_controller_ng/blob/10acca68afc6a166fa77420c3c4799ca5ccc10e2/bosh/jobs/cloud_controller_ng/spec#L610-L612).
You can setup thresholds in the [process monitor](https://github.com/DataDog/dd-agent/blob/master/conf.d/process.yaml.example#L25-L31).
Any chance we could integrate both?